sdk/resourcemanager/Proto.Client/src/Scenarios/CreateSingleVmExample.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Azure.ResourceManager.Core;
+using Azure.ResourceManager.Core;
```

Comment:
revert this file

Reply:
Not sure what change this is picking up, my local git doesn't show this as a diff

Reply:
just revert the file, could be a BOM thing